app/autoload.php

```diff
@@ -10,4 +12,6 @@
 
 AnnotationRegistry::registerLoader(array($loader, 'loadClass'));
 
+include_once __DIR__.'/bootstrap.php.cache';
```

Comment:
why force errors? its optional. even if it cannot be included, the app still works. no need to require it